linux_os/guide/services/ssh/ssh_server/var_sshd_set_maxstartups.var

```diff
@@ -0,0 +1,15 @@
+documentation_complete: true
+
+title: 'SSH MaxStartups setting'
+
+description: 'Configure parameters for maximum concurrent unauthenticatd connections to the SSH daemon.'
+
+type: string
+
+operator: equals
+
+interactive: true
+
+options:
+    default: "10:30:60"
```

Comment:
The sshd default is actually `10:30:100` -- perhaps leave `10:30:60` as an option and set it explicitly in the profile (like you already do)?